docker-compose.edx.yml

```diff
@@ -8,7 +8,7 @@ services:
     env_file: env.d/edx
     command: mysqld --character-set-server=utf8 --collation-server=utf8_general_ci
     networks:
-      potsie:
+      potsie_default:
```

Comment:
Not sure to understand the relation between this name and the one used in the commit description. I probably need more information IRL.

Reply:
By default, a network is created with potsie container and it is suffixed wirh `_default`. I can rename it in the commit for more clarity